app/serializers/user_card_serializer.rb

```diff
@@ -222,6 +223,10 @@ def card_background_upload_url
     object.card_background_upload&.url
   end
 
+  def status
```

Comment:
I think what we usually prefer to do is to define a `include_status?` method instead. Otherwise, we're going to be sending down `status: null` when the site setting is disabled.